site/oauth2-examples-oauth0.md

```diff
@@ -0,0 +1,134 @@
+# Use https://auth0.com/ as OAuth 2.0 server
+
+We are going to test 3 OAuth flows:
+1. Access management ui via a browser
+2. Access management rest api
+3. Access AMQP protocol
+
+## Prerequisites to follow this guide
+
+* Have an account in https://auth0.com/.
+* Docker
+
+## Create rabbitmq API
+
+In OAuth0, resources are mapped to Application APIs. Once you have logged onto your account in https://auth0.com/, go to dashboard > Applications > APIs > Create an API.
+
+* Give it the name `rabbitmq`. The important thing here is the `identifier` which must have the name of the *resource_server_id* we configured in RabbitMQ. This `identifier` goes into the `audience` JWT field. In our case, it is called `rabbitmq`
+* Choose `RS256` as the signing algorithm
+* Enable **RBAC**
+* Enable **Add Permissions in the Access Token**
+
```

Comment:
GLOBAL - same as previous comments:
Use numbered steps for procedures
Add full stops at the end of sentences
etc